sql/core/src/main/scala/org/apache/spark/sql/ContinuousQueryManager.scala

```diff
@@ -29,16 +29,16 @@ import org.apache.spark.sql.util.ContinuousQueryListener
 /**
  * :: Experimental ::
  * A class to manage all the [[org.apache.spark.sql.ContinuousQuery ContinuousQueries]] active
- * on a [[SQLContext]].
+ * on a [[SparkSession]].
  *
  * @since 2.0.0
  */
 @Experimental
-class ContinuousQueryManager(sqlContext: SQLContext) {
+class ContinuousQueryManager(sparkSession: SparkSession) {
```

Comment:
this is a public API, but it's not in any release yet so it's probably OK to change it.